package.json

```diff
@@ -23,6 +24,7 @@
     "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor",
```

Comment:
I think you can uninstall HR's linter since we're using the airbnb linter. Also, don't forget to update your STYLE-GUIDE.md to refer to the airbnb linter.